libcore/FileUtils.vala

```diff
@@ -194,14 +194,24 @@ namespace PF.FileUtils {
     /** Produce a valid unescaped path.  A current path can be provided and is used to get the scheme and
       * to interpret relative paths where necessary.
       **/
-    public string sanitize_path (string? p, string? cp = null, bool include_file_protocol = true) {
+    public string sanitize_path (string? _p, string? _cp = null, bool include_file_protocol = true) {
+        string p;
+        string cp;
```

Comment:
Can we use more descriptive variable names please? :)

Reply:
OK, done.